docs-md/tutorials/clickstream-docker.md

```diff
@@ -305,11 +305,11 @@ SELECT * FROM EVENTS_PER_MIN EMIT CHANGES LIMIT 5;
 Your output should resemble:
 
 ```
-1536662819576 | 24 : Window{start=1536662760000 end=-} | 24 | 12
-1536662819685 | 4 : Window{start=1536662760000 end=-} | 4 | 19
-1536662847582 | 4 : Window{start=1536662820000 end=-} | 4 | 75
-1536662847586 | 24 : Window{start=1536662820000 end=-} | 24 | 101
-1536662879959 | 29 : Window{start=1536662820000 end=-} | 29 | 2
+1536662819576 | 1536662760000 | 1536662765000 | 24 | 24 | 12
+1536662819685 | 1536662760000 | 1536662765000 | 4  | 4 | 19
+1536662847582 | 1536662820000 | 1536662825000 | 4  | 4 | 75
+1536662847586 | 1536662820000 | 1536662825000 | 24 | 24 | 101
+1536662879959 | 1536662820000 | 1536662825000 | 29 | 29 | 2
```

Comment:
Updated to reflect changes in `ROWKEY` and addition of `WINDOWSTART` and `WINDOWEND`.